content/en/docs/tasks/security/rbac-groups/index.md

```diff
@@ -147,99 +147,62 @@ communicating with `httpbin`.
 ## Configure groups-based authorization
 
 This section creates a policy to authorize the access to the `httpbin`
-service if the requests are originated from specific groups.
+workload if the requests are originated from specific groups.
 As there may be some delays due to caching and other propagation overhead,
-wait until the newly defined RBAC policy to take effect.
+wait until the newly defined authorization policy to take effect.
 
-1.  Enable the Istio RBAC for the namespace:
+1. Run the following command to create a `deny-all` policy in the `default` namespace.
+   The policy doesn't have a `selector` field, which applies the policy to every workload in the
+   `$NS` namespace. The `spec:` field of the policy has the empty value `{}`.
+   That value means that no traffic is permitted, effectively denying all requests.
 
     {{< text bash >}}
     $ cat <<EOF | kubectl apply -n $NS -f -
-    apiVersion: "rbac.istio.io/v1alpha1"
-    kind: ClusterRbacConfig
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
     metadata:
-      name: default
+      name: deny-all
     spec:
-      mode: 'ON_WITH_INCLUSION'
-      inclusion:
-        namespaces: ["rbac-groups-test-ns"]
+      {}
     EOF
     {{< /text >}}
 
-1.  Once the RBAC policy takes effect, verify that Istio rejected the curl
-connection to the `httpbin` service:
+1.  Once the policy takes effect, verify that Istio rejected the curl
+connection to the `httpbin` workload:
 
     {{< text bash >}}
     $ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
     {{< /text >}}
 
-    Once the RBAC policy takes effect, the command returns the HTTP code 403.
+    Once the policy takes effect, the command returns the HTTP code 403.
 
-1.  To give read access to the `httpbin` service, create the `httpbin-viewer`
-service role:
+1.  To give read access to the `httpbin` workload, create the `httpbin-viewer`
```

Comment:
To help readers understand better, consider adding an explanation of workload-selector based policy enforcement, e.g., replace "To give read access to the `httpbin` workload" with the following:
To give read access to the `httpbin` workload, which is selected through the label `app: httpbin`, ....